bigtable/client/idempotent_mutation_policy_test.cc

```diff
@@ -0,0 +1,31 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/client/idempotent_mutation_policy.h"
+
+#include <gmock/gmock.h>
```

Comment:
Not sure if you care: this test doesn't seem to use any mocking, so `gtest.h` should suffice for the `EXPECT_(TRUE|FALSE)` checks.

I haven't noticed if you just `#include <gmock/gmock.h>` in every test for consistency; I am guessing GMock includes all of GTest, so this is a superset everywhere. Not a big deal, just curious.

Reply:
Consistency due to laziness?  As far as I know, gmock is a superset, and there are some matches in gmock that can be very useful, e.g., `testing::ElementsAre()` is defined in gmock <shrug>